src/main/java/org/broadinstitute/hellbender/tools/spark/ApplyBQSRSpark.java

```diff
@@ -48,15 +49,22 @@
     @Argument(doc = "If specified, shard the output bam", shortName = "shardedOutput", fullName = "shardedOutput", optional = true)
     private boolean shardedOutput = false;
```

Comment:
Move `shardedOutput` up to `GATKSparkTool` as well -- as with `parallelism` we can make it clear in the docs that it's not applicable to all tools.